### ISSUE TITLE
azure-identity package from PyPi not buildable from source

### ISSUE THREAD

**Issue by glaubitz** [Azure/azure-sdk-for-python#7526]
The ```azure-identity``` package that is currently distributed on PyPi is not buildable from source:

```
glaubitz@suse-laptop:/tmp/azure-identity> unzip azure-identity-1.0.0b3.zip 
Archive:  azure-identity-1.0.0b3.zip
   creating: azure-identity-1.0.0b3/
   creating: azure-identity-1.0.0b3/azure_identity.egg-info/
   creating: azure-identity-1.0.0b3/azure/
  inflating: azure-identity-1.0.0b3/PKG-INFO  
 (...)
  inflating: azure-identity-1.0.0b3/azure/identity/aio/__init__.py  
  inflating: azure-identity-1.0.0b3/azure/identity/aio/_managed_identity.py  
  inflating: azure-identity-1.0.0b3/azure/identity/aio/credentials.py  
  inflating: azure-identity-1.0.0b3/azure/identity/aio/_internal/exception_wrapper.py  
  inflating: azure-identity-1.0.0b3/azure/identity/aio/_internal/__init__.py  
glaubitz@suse-laptop:/tmp/azure-identity> cd azure-identity-1.0.0b3/
glaubitz@suse-laptop:/tmp/azure-identity/azure-identity-1.0.0b3> python3.7 setup.py build
Traceback (most recent call last):
  File "setup.py", line 40, in <module>
    with open("HISTORY.md", encoding="utf-8") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'HISTORY.md'
glaubitz@suse-laptop:/tmp/azure-identity/azure-identity-1.0.0b3>
```

After manually copying ```HISTORY.md``` from git, the build still doesn't proceed:

```
glaubitz@suse-laptop:/tmp/azure-identity/azure-identity-1.0.0b3> cp -av ../../azure-sdk-for-python/sdk/identity/azure-identity/HISTORY.md .
'../../azure-sdk-for-python/sdk/identity/azure-identity/HISTORY.md' -> './HISTORY.md'
glaubitz@suse-laptop:/tmp/azure-identity/azure-identity-1.0.0b3> python3.7 setup.py build
running build
glaubitz@suse-laptop:/tmp/azure-identity/azure-identity-1.0.0b3>
```

**Comment by maggiepint** [comment]
Assuming this is related to #7524 but see that it is somewhat different.

@scbedd again, and maybe @johanste - perhaps it's time to visit our python local build story?

**Comment by scbedd** [comment]
Hey @glaubitz, thanks for the report! 

Absolutely related to #7524 Maggie

It's actually the same issue #7494 and it's fixed by the same [commit](https://github.com/Azure/azure-sdk-for-python/commit/ab3f145f1cebafa8ed4e533073ad7eaf75b37108#diff-2eac4e3cd68e73f8d6fb0f2039669508).

**Comment by chlowell** [comment]
The published 1.0.0b4 source builds so I'm closing this as fixed, thank you for opening it.
